src/topicButton.js

```diff
@@ -18,12 +18,26 @@ var TopicButton = GObject.registerClass({
 }, class TopicButton extends Gtk.Frame {
     _init(props = {}) {
         super._init(props);
+        this._needsAttention = false;
 
         const overlay = new Gtk.Overlay();
         const box = new Gtk.Box({
             halign: Gtk.Align.FILL,
             orientation: Gtk.Orientation.VERTICAL,
         });
+        this._attentionSign = new Gtk.Revealer({
```

Comment:
This doesn't currently have a transition type set, are you waiting for a spec on that?

Reply:
Yes. I left it as the default for that reason.